workers/unity/Packages/com.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -14,13 +15,10 @@ public static class LocalLaunch
         private const string InspectorUrl = "http://localhost:21000/inspector";
 
         private static readonly string
-            SpatialProjectRootDir = Path.GetFullPath(Path.Combine(Application.dataPath, "..", "..", ".."));
+            DefaultLogFileName = Path.GetFullPath(Path.Combine(Common.SpatialProjectRootDir, "*unityclient.log"));
```

Comment:
Why is there a * in front of the DefaultLogFileName?
Also, is Unityclient the best default?
Also, why put the log file in the root of the project, increasing clutter?

Reply:
Just pinging again about the location of the log file.

Reply:
as mentioned above: the * is used in a regex later on to search for this log file from the root folder, but it doesn't have to be in the root.

Reply:
My complaint is the fact that it's dumped into the root directory :)